external/thorntail-mp-tck/playlist.xml

```diff
@@ -12,16 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 -->
-<!-- Tests require a larger java heap than the default when running with openj9 jdk.  -->
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TKG/playlist.xsd">
 	<test>
 		<testCaseName>thorntail_microprofile_tck</testCaseName>
-		<command>docker run --name thorntail-mp-tck --env OPENJ9_JAVA_OPTIONS=-Xmx1g $(EXTRA_DOCKER_ARGS) adoptopenjdk-thorntail-mp-tck:latest \
-			 "-pl testsuite/microprofile-tcks,testsuite/microprofile-tcks/config,testsuite/microprofile-tcks/fault-tolerance,testsuite/microprofile-tcks/health,testsuite/microprofile-tcks/jwt-auth,testsuite/microprofile-tcks/metrics,testsuite/microprofile-tcks/openapi,testsuite/microprofile-tcks/opentracing,testsuite/microprofile-tcks/restclient"; \
-			 $(TEST_STATUS); \
-			 docker cp thorntail-mp-tck:/testResults/surefire-reports $(REPORTDIR)/external_test_reports; \
-			 docker rm -f thorntail-mp-tck; \
-			 docker rmi -f adoptopenjdk-thorntail-mp-tck
+		<command> $(TEST_ROOT)$(D)external$(D)external.sh --run --tag ${DOCKERIMAGE_TAG} --version ${JDK_VERSION} --impl ${JDK_IMPL} --dir thorntail-mp-tck --reportsrc /testResults/surefire-reports --reportdst $(REPORTDIR) --docker_args $(EXTRA_DOCKER_ARGS) \
+		--testtarget "-pl testsuite/microprofile-tcks,testsuite/microprofile-tcks/config,testsuite/microprofile-tcks/fault-tolerance,testsuite/microprofile-tcks/health,testsuite/microprofile-tcks/jwt-auth,testsuite/microprofile-tcks/metrics,testsuite/microprofile-tcks/openapi,testsuite/microprofile-tcks/opentracing,testsuite/microprofile-tcks/restclient" ; \
```

Comment:
Can you move the --testtarget "-pl xxxx" before the --docker_args $(EXTRA_DOCKER_ARGS) - I believe we are hitting a problem with external.sh, it expects --docker_args to be the last option passed in (otherwise it will assume the remaining arguments are the EXTRA_DOCKER_ARGS